runners/apex/src/main/java/org/apache/beam/runners/apex/translation/ParDoBoundTranslator.java

```diff
@@ -43,6 +43,10 @@ public void translate(ParDo.Bound<InputT, OutputT> transform, TranslationContext
     DoFn<InputT, OutputT> doFn = transform.getFn();
     DoFnSignature signature = DoFnSignatures.getSignature(doFn.getClass());
 
+    if (signature.processElement().isSplittable()) {
+      throw new UnsupportedOperationException(
+          String.format("ApexRunner does not support Splittable DoFn: %s", doFn));
```

Comment:
Error message nits:

 - For proper identifiers, interpolate when possible, as in `ApexRunner.class.getSimpleName()`
 - `Splittable` shouldn't be capitalized. It is just a `splittable DoFn`.